### ISSUE TITLE
Site wide

### ISSUE THREAD

**Issue by getgray** [jnorell/train-spam-scanner#1]
Is this compatible with a site-wide bayes?  I don't have individual user IMAP spam/ham folders.

**Comment by jnorell** [comment]
Yes, it is designed for use with site-wide bayes, not per-user bayes (otherwise you would train every user's bayes database identically, with the site-wide training corpus, so makes no sense).
